packages/login/network/settings.js

```diff
@@ -50,6 +55,7 @@ const MAINNET_BASE = {
   scanner: blockExplorersMap.Ethereum.mainnet,
   [BLOCKCHAIN_BITCOIN]: 'bitcoin',
   [BLOCKCHAIN_BITCOIN_CASH]: 'bitcoin',
+  [BLOCKCHAIN_DASH]: 'bitcoin',
```

Comment:
Is it correct?

Reply:
It is just an identifier. Everywhere in the source code we detect the network type comparing with 'test' i.e. it is required that the testnet identifier should be 'testnet'. The mannet identifiers can be anything but not 'testnet'